src/screens/DateFilterScreen/DateRangePickerDay.js

```diff
@@ -38,17 +38,17 @@ type DayMarking = {
 };
 
 type DayProps = {
-  state?: "disabled",
+  disabled: boolean,
```

Comment:
I think you can get rid of `disabled` here now that you calculate it based on todays date. You can then also remove the defaultProps.